packages/deployer/test/integration/helpers/use-environment.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const path = require('path');
+const { TASK_DEPLOY } = require('@synthetixio/deployer/task-names');
+const { defaultDeploymentInfo } = require('../../../utils/deployments');
+const { resetHardhatContext } = require('hardhat/plugins-testing');
+
+function loadEnvironment(fixtureProjectName) {
+  resetHardhatContext();
+
+  const envPath = _getEnvironmentPath(fixtureProjectName);
+  process.chdir(envPath);
+
+  return require('hardhat');
+}
+
+async function deployOnEnvironment(hre, customOptions = {}) {
+  const deploymentInfo = {
+    ...defaultDeploymentInfo,
+    proxyContract: hre.config.deployer.proxyContract,
+    network: hre.config.defaultNetwork,
+    instance: 'test',
+  };
+
+  await hre.run(TASK_DEPLOY, {
+    ...deploymentInfo,
+    noConfirm: true,
+    quiet: true,
+    ...customOptions,
+  });
+
+  if (customOptions.clear) {
+    try {
+      const initializer = require(path.join(
+        hre.config.paths.root,
+        'test',
+        'helpers',
+        'initializer'
+      ));
+
+      await initializer(deploymentInfo, hre);
+    } catch (err) {
+      // eslint-disable-next-line
+    }
+  }
```

Comment:
This is so that we don't "eat" another errors, and also not include the initializer inside the try catch:

```suggestion
    } catch (err) {
      if (err.code === 'MODULE_NOT_FOUND') {
        return defaults;
      }

      throw err;
    }

    await initializer(deploymentInfo, hre);
  }
```